### ISSUE TITLE
How to print level

### ISSUE THREAD

**Issue by shamim96** [Arkitecht/fedex-laravel#6]
Hi,
I have a website where customer can sell their old phone. After a customer desire to sell their phone, they will come to my website and select their phone and go for checkout. when they will checkout they will get the print copy of fedex lebel so they can send their phone using fedex. 
If you use your this package can i get this feature ? How could i print the lebel from response?

**Comment by syedMSohaib** [comment]
Are you getting the response. can you send me the sample code ? , i am getting difficulties !

**Comment by shamim96** [comment]
I did not get the response. If you get any solution please tell me.

**Comment by syedMSohaib** [comment]
Nope, I didn't , now using rocketship, but do look at this project may be it should help you.
[https://github.com/GabrielKobus0622/Fedex_USHIP_UPS_FREIGHTQUOTE_Laravel_Project](url)
